handbook/product/index.md

```diff
@@ -18,12 +18,15 @@ You can reach us at the #product channel or @product-team on Slack. If you have
 
 ## Metrics
 
+- [User metric definitions](../ops/bizops/user_definitions.md)
 - [Product metrics dashboard](https://sourcegraph.looker.com/dashboards/127)
 - [How to add metrics](../ops/bizops/analytics.md#How-to)
 
 ## Goals
 
-Within the product organization, individual product engineering teams set their own goals.
+We have a topline goal of 10k [monthly active users](../ops/bizops/user_definitions.md) on the cloud platform for FY22.
```

Comment:
Is everyone comfortable with explicitly setting this goal? Are there other components to the goal, or additional context to record?

Reply:
We can instead point at the OKR pages once they are updated rather than embedding the goal here.

Reply:
That makes sense. I'll keep this discussion open until we make a decision, then once it's in the OKR and linked to from here I'll resolve this thread.

Reply:
@christinaforney I believe this is the last item for this first iteration - two questions:

- In terms of goal setting, do you already have something in mind for updating the OKRs?
- Are you good to go with the rest of the changes in this PR?

Thanks!